src/genopheno.h

```diff
@@ -100,13 +106,20 @@ namespace libcmaes
       if (!_id)
 	{
 	  dMat ncandidates = dMat(candidates.rows(),candidates.cols());
+#ifdef HAVE_TBB
+    tbb::parallel_for(size_t(0), size_t(candidates.cols()), size_t(1), [&](size_t i) {
+#else
```

Comment:
These definitions should go in a cpp file.

Reply:
I am expecting @beniz opinion on this, because as I already said: These definitions **CANNOT** go to a cpp file; the genopheno class is templated! So either we remove TBB completely from this file (and have no TBB parallelization there) or we leave it and put an explicit remark for users to defive HAVE_TBB if they want TBB. I am OK with both..